populate_db.sql

```diff
@@ -10,22 +10,23 @@
 
 -- with
 -- token_times as (
---     select 
+--     select
 --         date_trunc('hour', evt_block_time) as hour,
---         date_trunc('minute', evt_block_time) as minute, 
---         contract_address 
+--         date_trunc('minute', evt_block_time) as minute,
+--         contract_address
 --     from erc20."ERC20_evt_Transfer"
 --     where '\x9008D19f58AAbD9eD0D60971565AA8510560ab41' in ("to", "from")
 --     and evt_block_time between '2022-03-01' and '2022-03-11'
+--     group by hour, minute, contract_address
 -- )
```

Comment:
Seed data query was returning contract_address (i.e. tokens) at specific times with multiplicity, resulting in duplicates prices for tokens. This trimmed down the size of the prices tables quite a lot.